website/docs/developer/research/research-deep-glacier-migration.md

```diff
@@ -0,0 +1,154 @@
+---
+id: research-deep-glacier-migration
+title: Deep archive storage migration Research Notes
+description: Research Notes on deep archive migration.
+---
+
+## Overview
+
+Deep archive S3 storage type is the cheapest glacier storage type currently present in AWS S3. For archive data that does not require immediate access such as backup or disaster recovery use cases, choosing `Deep Archive Access` tier could be a wise choice.
+- Storage cost is ~$1/TB data per month. 
+- Objects in the Deep Archive Access tier are moved to the Frequent Access tier within 12 hours. Once the object is in the Frequent Access tier, users can send a GET request to retrieve the object.
```

Comment:
"are moved" implies it happens automatically. Recommend "can be moved".

Reply:
fixed